README.md

```diff
@@ -138,13 +135,22 @@ options = {
    },
    plugins: {
       chromecast: {
-         buttonText: 'Chromecast', // Not required
+         receiverAppID: '1234' // Not required
       },
-      receiverAppID: '1234' // Not required
    }
 };
 ```
 
+##### Localization
+
+The `ChromecastButton` component has one translated string: "Open Chromecast menu". The
+"Open Chromecast menu" string appears in both of the standard places for Button component
+accessibility text: inside the `.vjs-control-text` span and as the `<button>` element's
+`title` attribute.
+
+To localize the Chromecast button text, follow the steps in the
+[Video.js Languages tutorial][videojs-docs] to add a `"Open Chromecast menu"` key to the map
```

Comment:
to ad **an** `"Open Chromecast menu"` key ...

Reply:
Good catch, thanks!